.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "cpp/submodules/parquet-testing"]
-	path = cpp/submodules/parquet-testing
-	url = https://github.com/apache/parquet-testing.git
```

Comment:
Did you mean to do this? I think it's causing the spurious test failures

Reply:
@jbonofre I think the build failures here are solely because the submodules have been removed. If we revert that then I can finish the donation paperwork.

Reply:
I think it was not intentional 😄 
Sorry about that. Let me fix that.

Reply:
+1 to David, I see the [CI build](https://github.com/apache/arrow/actions/runs/14362316261/job/40266700320?pr=40939#step:10:12911) is complaining that `parquet` tests are failing because they are removed. If those lines are added back, the `parquet` tests should pass.